tests/integration/inventory.networking

```diff
@@ -0,0 +1,30 @@
+[nx_os]
```

Comment:
@AAYUSH2091 plz remove this file, we dont push inventory file

Reply:
done